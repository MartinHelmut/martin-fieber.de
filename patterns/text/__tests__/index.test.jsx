import React from "react";
import renderer from "react-test-renderer";

import Text from "../index";

describe("Text", () => {
  test("should render", () => {
    const tree = renderer.create(<Text>Regular text ...</Text>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
