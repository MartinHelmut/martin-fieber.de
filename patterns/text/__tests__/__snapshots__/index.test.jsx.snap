// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text should render 1`] = `
.c0 {
  font-family: var(--base-font);
  font-size: 1.4rem;
  font-weight: 300;
  line-height: 1.65;
  margin: 0 0 var(--size-500) 0;
  padding: 0;
}

<p
  className="c0"
>
  Regular text ...
</p>
`;
